src/kOS/Suffixed/WaypointValue.cs

```diff
@@ -8,14 +8,14 @@ namespace kOS.Suffixed
 {
```

Comment:
All of the changes to this file were done to avoid non private fields and fix a spelling mistake. i am sneaking it in here, because im bad.